test/e2e/tests/custom-rpc-history.spec.js

```diff
@@ -70,7 +70,7 @@ describe('Stores custom RPC history', function () {
           '.networks-tab__add-network-form-footer .btn-primary',
         );
 
-        await driver.findElement({ text: networkName, tag: 'p' });
+        await driver.findElement({ text: networkName, tag: 'span' });
```

Comment:
Changed the wrapping html element of the text from a `p` to `span` tag in the `PickerNetwork`